checks/http/mhcsoftwareinc.rb

```diff
@@ -7,16 +7,16 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'application',
-              tags: %w[COTS WebServer],
+              tags: ['COTS', 'Web Server'],
               vendor: 'MHCSoftwareInc',
               product: 'Document Self-Service',
               references: ['https://www.mhcsoftwareinc.com/platform/application-suites/document-self-service/'],
               version: nil,
               description: 'MHCSoftwareInc Document Self-Service - Server Header',
               match_type: :content_headers,
-              match_content: %r{^server:\ DSS WebServer/(\d\.\d)$}i,
+              match_content: %r{^server:\ DSS Web Server/(\d\.\d)$}i,
```

Comment:
@chowdud Can you check if this fingerprint has been broken with the find/replace ?

Reply:
Reverting this back to the original.